cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -631,7 +632,11 @@ func registerChecks(wlm workloadmeta.Component, tagger tagger.Component, cfg con
 	corecheckLoader.RegisterCheck(kubernetesapiserver.CheckName, kubernetesapiserver.Factory(tagger))
 	corecheckLoader.RegisterCheck(ksm.CheckName, ksm.Factory())
 	corecheckLoader.RegisterCheck(helm.CheckName, helm.Factory())
-	corecheckLoader.RegisterCheck(disk.CheckName, disk.Factory())
+	if cfg.GetBool("use_diskv2_check") {
```

Comment:
I know it was already there but I'm a bit confused by this one because there's not Python interpreter in the cluster-agent. The disk check is not run according to the `status` command.

Reply:
To be honest, I don't know if it runs by default on the cluster-agent or not, but in the worst case this change would keep the current behavior since `use_diskv2_check` is false by default. What do you think?